### ISSUE TITLE
Why is there darwin-x64-49/binding.node this version?

### ISSUE THREAD

**Issue by F-happy** [electron/electron#7008]
* Electron version: 1.3.4
* Operating system: 10.11.6

I use electron to build my project, and the introduction of node-sass tips I need "node-sass/vendor/Darwin - x64-49 / binding", but I'm in the release of the node-sass at most only 48, why will appear this kind of circumstance?

**Comment by F-happy** [comment]
this here 
<img width="435" alt="qq20160827-0 2x" src="https://cloud.githubusercontent.com/assets/8241236/18028357/3c2e2cec-6caf-11e6-9569-b336adfa9d8d.png">
i perform  'sass.getBinaryPath() ', return '/node_modules/node-sass/vendor/darwin-x64-49/binding.node',
but, this should '/node_modules/node-sass/vendor/darwin-x64-48/binding.node'.....
<img width="691" alt="qq20160827-1 2x" src="https://cloud.githubusercontent.com/assets/8241236/18028371/c76cd178-6caf-11e6-800b-77b14c5182b7.png">

**Comment by F-happy** [comment]
Thank you, I passed the document to rebuild the node-sass, but still can not run, so I still go to the fix in the meantime....or reduce the electron version. @enlight

**Comment by zcbenz** [comment]
I'm closing this since it is a question on building native modules, for further questions I suggest seeking help from [the community](https://github.com/electron/electron#community).

**Comment by littledu** [comment]
I have the same question.

![image](https://cloud.githubusercontent.com/assets/1784673/18407751/849eb36e-774c-11e6-8f7d-072156e588a0.png)

- Electron version: 1.3.5
- Node version: 6.3.0

 I passed the document to rebuild the node-sass, but still can not run. I can't find a solution for this question. So sad.

**Comment by F-happy** [comment]
这个问题你可以使用低版本的 electron 来解决，我测试在1.2.8版本上就可以正常使用了，目前我也没有找到什么好的方式，你可以试试1.2.8版本。 @littledu

**Comment by enlight** [comment]
Yeah, so, `node-sass` has hard-coded the (`supported runtimes`)[https://github.com/sass/node-sass/blob/v3.9.3/lib/extensions.js#L83] so someone will have to submit a PR to update it to support the recent Electron versions. Fortunately you should be able to bypass that runtime check by setting the path to the rebuilt `node-sass` binary. If you look at [how `node-sass` searches for the binary](https://github.com/sass/node-sass/blob/v3.9.3/lib/extensions.js#L243) you'll see you can either set the `SASS_BINARY_PATH` env var or the `npm_config_sass_binary_path` NPM config var to point to the rebuilt `.node` binary.

**Comment by littledu** [comment]
@F-happy  谢谢，我是在 MAC 下，用 1.2.8 的话，都不需要重新编译，但如果是 Windows 下则需要，之前用的 0.37.8 可以 rebuild 通过，我想 1.2.8 应该也可以。

**Comment by littledu** [comment]
Thanks @enlight
